recipes/spritz/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.3.3" %}
+
+package:
+  name: spritz
+  version: {{ version }}
+
+source:
+  url: https://github.com/smith-chem-wisc/Spritz/releases/download/{{ version }}/SpritzCMD.zip
+  sha256: f218e3ebb599b29960ae0245aa4a16ab8734ed7895d98f0772b1f7734dacfc10
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - sysroot_linux-64 2.17  # [linux64]
+  run:
+    - snakemake-minimal =5.27.4
+    - mamba
```

Comment:
mamba and conda can be removed afaik, or are you installing dependencies during tool runtime?

Reply:
Yes, Spritz installs dependencies during tool runtime using snakemake with conda/mamba.

Reply:
Why does needs to happen during tool runtime? Please be aware that cluster installations exists, where conda is read-only from worker nodes.

Reply:
Spritz has three workflows right now, and not all tools are required if users just need one of the three. In my tests, this is more efficient, not least because spritz uses mamba and users tend to use conda installs by default. Spritz has an option for protected-access cluster setups, where resources (e.g. uniprot, ensembl databases) are downloaded beforehand. I believe snakemake, which is part of the backend of spritz, sets up environments (e.g. using pre-downloaded bioconda/conda-forge repositories) and then activates them before firing up worker nodes. I've tested this via `slurm` on a cluster available to me. Are there other cluster setups that are problematic here?

Reply:
Thanks for the heads-up on the possible cluster installation issues, by the way. I'm happy to do some more testing of that issue before merging if that would be best and if I can find such an environment.

Reply:
All I can say is that installation of packages during runtime is not usual, a dependency on conda/mamba as runtime dependency is not usual. Why not splitting up your workflows into 3 packages? We don't have strict rules here about that I just wanted to point it out and this might cause problems down the road for your users.

Reply:
Makes sense!

The workflows aren't separate; they can be combined. For example, users can make databases with variants or transcript isoforms, or they can request both variants and isoforms together which yields a different result than they do separately. Similarly, users can request to do quantification of these features or not.

Reply:
But Bioconda is not a workflow repository but a tool repository :)
I know the lines a blurry, but this is the reason repos and communities like snakepipes, nf-core etc exists. Workflows are a different beast.

Reply:
Oh that's an interesting point. Snakepipes does look like it might be a more appropriate home for Spritz.